binance_trade_bot/binance_api_manager.py

```diff
@@ -95,7 +96,7 @@ def get_ticker_price(self, ticker_symbol: str):
             self.logger.debug(f"Fetched all ticker prices: {self.cache.ticker_values}")
             price = self.cache.ticker_values.get(ticker_symbol, None)
             if price is None:
-                self.logger.debug(f"Ticker does not exist: {ticker_symbol} - will not be fetched from now on")
+                self.logger.info(f"Ticker does not exist: {ticker_symbol} - will not be fetched from now on")
```

Comment:
Minor: I changed this one to debug because it's annoying to have the telegram notification every time the bot restarts, it's usually just USDTBTC